.github/workflows/compliance.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       - uses: mtfoley/pr-compliance-action@main
         with:
           body-auto-close: false
+          body-regex: (fix|resolv|clos|r)(e)*(s|d)* (#[1-9]+\d*|\S+\/\S+\/issues\/[1-9]+\d*)
```

Comment:
> disallow leading zeros on issue numbers

Is this a "bug" in `mtfoley/pr-compliance-action`, that it allows leading zeros? If so, I'd say it'd be best to file it on https://github.com/mtfoley/pr-compliance-action.

Reply:
The default regex checks `#\d+` for issues. `#[1-9]+\d*` should disallow leading zeros.
I'll open an issue over there for this.

Reply:
https://github.com/mtfoley/pr-compliance-action/issues/346